data/kernels/blendop.cl

```diff
@@ -21,8 +21,8 @@
 
 typedef enum dt_develop_blend_mode_t
 {
-  DEVELOP_BLEND_DISABLED = 0x00,
-  DEVELOP_BLEND_NORMAL = 0x01, /* deprecated as it did clamping */
+  /* DEVELOP_BLEND_DISABLED = 0x00, same as the new normal */
+  /* DEVELOP_BLEND_NORMAL = 0x01, deprecated as it did clamping */
```

Comment:
Why not define the values here and use them in _blend_legacy_blend_mode rather than having magic numbers there? If you want to make sure they are not used accidentally anywhere else, add _DEPRECATED to the name in both locations.

Reply:
Ok for me, done.